openbb_platform/providers/polygon/openbb_polygon/models/cash_flow.py

```diff
@@ -113,6 +119,7 @@ def extract_data(
         api_key = credentials.get("polygon_api_key") if credentials else ""
 
         base_url = "https://api.polygon.io/vX/reference/financials"
+        query.period = "quarterly" if query.period == "quarter" else query.period
```

Comment:
Key Error on cashflow statement.

```
In [11]: obb.stocks.fa.cash("AAPL", provider="polygon", period= "ttm").to_df().transpose()
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
File ~/GitHub/OpenBBTerminal/openbb_platform/platform/provider/openbb_provider/query_executor.py:87, in QueryExecutor.execute(self, provider_name, model_name, params, credentials, **kwargs)
     86 try:
---> 87     return fetcher.fetch_data(params, filtered_credentials, **kwargs)
     88 except Exception as e:

File ~/GitHub/OpenBBTerminal/openbb_platform/platform/provider/openbb_provider/abstract/fetcher.py:55, in Fetcher.fetch_data(cls, params, credentials, **kwargs)
     54 data = cls.extract_data(query=query, credentials=credentials, **kwargs)
---> 55 return cls.transform_data(query=query, data=data, **kwargs)

File ~/GitHub/OpenBBTerminal/openbb_platform/providers/polygon/openbb_polygon/models/cash_flow.py:143, in PolygonCashFlowStatementFetcher.transform_data(query, data, **kwargs)
    140 for item in data:
    141     sub_data = {
    142         key: value["value"]
--> 143         for key, value in item["financials"]["cash_flow_statement"].items()
    144     }
    145     sub_data["date"] = item["start_date"]

KeyError: 'cash_flow_statement'
```

Reply:
`ttm` is only for the balance sheet. Removing these from Polygon for cash flow and income statement.

![image](https://github.com/OpenBB-finance/OpenBBTerminal/assets/88149434/88aaf36d-f5a8-4738-8738-a4438fdba9ce)